twolame.pc.in

```diff
@@ -8,4 +8,4 @@ Description: An optimized MPEG Audio Layer 2 encoder
 Requires:
 Version: @VERSION@
 Libs: -L${libdir} -ltwolame
-Cflags: -I${includedir}
+Cflags:  @CFLAGS@ -I${includedir}
```

Comment:
I could have sworn the behavior was different the last time I checked, but yeah, that's not desirable.  I think I've found a slightly different solution, which requires less.